submissions/ses3332002/dom/script.js

```diff
@@ -0,0 +1,260 @@
+"use strict";
+window.onload = function() {
+
+// data for displaing
+  const alboms = [
```

Comment:
It's a typo here.
```suggestion
  const albums = [
```

Reply:
My mistake :)
I'll remake it.

Reply:
BTW, in order to avoid such mistakes, you can install a spell checker.
As an example for VSC you can use this - https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
ready